blocks/components/editable/index.js

```diff
@@ -112,11 +111,18 @@ export default class Editable extends wp.element.Component {
 	}
 
 	getRelativePosition( node ) {
-		const editorPosition = this.editor.getBody().closest( '.editor-visual-editor__block' ).getBoundingClientRect();
 		const position = node.getBoundingClientRect();
+
+		// Find the parent "relative" positioned container
+		const container = this.props.inlineToolbar
+			? this.editor.getBody().closest( '.blocks-editable' )
```

Comment:
Should we just assign a `ref` to the `.blocks-editable` node rendered by this component? I'd worried we were making more assumptions about surrounding content. I still think we should look to eliminate traversing to `.editor-visual-editor__block` but I'm not going to worry about it a whole lot here since it already exists.